infobip-spring-data-jdbc-querydsl/src/test/java/com/infobip/spring/data/jdbc/NoArgsEntity.java

```diff
@@ -23,7 +23,6 @@
         this.value = null;
     }
 
-    @PersistenceConstructor
```

Comment:
reverts the change that I needed to make in https://github.com/infobip/infobip-spring-data-querydsl/pull/37 (so removing this proves that it's backwards compatible)